### PR TITLE
[GCI] Standardised Blob Analysis module comments

### LINKED ISSUE #1350: Standardise Blob Analysis module comments
Note: **We are preparing to participate in Google Code-in, and have reserved this issue for participants in GCI - but we'd love to have your help with another one! Please check out https://code.publiclab.org to see more.**

This is a sub-issue of #1046

The comments in the following files are inconsistent:
1. [Blur.js](https://github.com/publiclab/image-sequencer/blob/main/src/modules/BlobAnalysis/BlobAnalysis.js)
2. [Module.js](https://github.com/publiclab/image-sequencer/blob/main/src/modules/BlobAnalysis/Module.js)

Rules:
1. Methods must be described using [JSDoc comments](https://devdocs.io/jsdoc/)
2. Misc code comments should be inline unless it is a long sentence.
3. No use of continuous tenses, no pronouns.
4. No redundant comments.
5. Each comment should start with an uppercase letter and end with a full stop.

### PR DESCRIPTION
**Pull request by HarshKhandeparkar**
Fixes #1350  (<=== Replace `0000` with the Issue Number)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below
* [x] Insert-step functionality is working correct as expected.
> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
Please make sure to get at least two reviews before asking for merging the PR as that would make the PR more reliable on our part
Thanks!

### CODE REVIEW

**Comment by HarshKhandeparkar** [comment]
@SidharthBansal @jywarren here's the last PR with two more file changes. Please try to merge this soon as the deadline is only 2 days away. Thanks a lot to all!

File: src/modules/BlobAnalysis/BlobAnalysis.js [hunk 1]
```diff
@@ -25,26 +25,26 @@ module.exports = function(pixels, options, priorStep){
   let unknown = new cv.Mat();
   let markers = new cv.Mat();
  
-  // gray and threshold image
+  // Gray and Threshold the image image.
```
Comment (SidharthBansal) [inline src/modules/BlobAnalysis/BlobAnalysis.js]: Why Threshold with `T` instead of `t`?
Any reason for 2 `image`?

**Comment by SidharthBansal** [comment]
Tip: start opening ftos to get points.
Documentation from will contain points too.
If you are free you can do that. :-)
Don't write diff. Instead ask them what good documentation needs. And help them to implement good docs.
Don't provide diff. That is no learning from my point of view. Especially in case of documentation 
Thanks 😊

**Comment by SidharthBansal** [comment]
Please don't create all documentation ftos in a single day. That will cause a flood of documentation and we need to close. So limit it to 1-2 per day.
However you can create as many non documentation ftos as you want per day in any repo.
Also, you can create 1-2 documentation ftos per day in other repos if you are familiar with the code.
Thanks

File: src/modules/BlobAnalysis/BlobAnalysis.js (hunk 1 above)
Comment (HarshKhandeparkar) [reply src/modules/BlobAnalysis/BlobAnalysis.js]: No reason for 2 images but I think Thresholding is a proper name. I'll fix the doubl img.

Comment (HarshKhandeparkar) [inline src/modules/BlobAnalysis/BlobAnalysis.js]:
```suggestion
  // Gray and Threshold the image.
```

**Comment by HarshKhandeparkar** [comment]
I know. I am going to open a few docs FTOs once this task is over. TY.

**Comment by HarshKhandeparkar** [comment]
Have you read the sub-issues I opened? Do you think it will be good if I make more such issues but in the FTO format for GCI?

**Comment by SidharthBansal** [comment]
Commented on it.
I asked you to open the other ftos so that you will remain not be blocked
due to lesser availability of mentors. This year gci is epic so it is
taking a lot of time.

File: src/modules/BlobAnalysis/BlobAnalysis.js [hunk 2]
```diff
@@ -25,26 +25,26 @@ module.exports = function(pixels, options, priorStep){
   let unknown = new cv.Mat();
   let markers = new cv.Mat();
  
-  // gray and threshold image
+  // Gray and Threshold the image.
   cv.cvtColor(src, gray, cv.COLOR_RGBA2GRAY, 0);
```
Comment (SidharthBansal) [inline src/modules/BlobAnalysis/BlobAnalysis.js]: threshold. 

Pls remove . At end of phrases

File: src/modules/BlobAnalysis/BlobAnalysis.js [hunk 3]
```diff
@@ -25,26 +25,26 @@ module.exports = function(pixels, options, priorStep){
   let unknown = new cv.Mat();
   let markers = new cv.Mat();
  
-  // gray and threshold image
+  // Gray and Threshold the image.
```
Comment (HarshKhandeparkar) [inline src/modules/BlobAnalysis/BlobAnalysis.js]:
```suggestion
  // Gray and Threshold the image
```

File: src/modules/BlobAnalysis/BlobAnalysis.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 module.exports = function(pixels, options, priorStep){
 
-  var $ = require('jquery'); // to make Blob-analysis work for node.js
+  var $ = require('jquery'); // To make Blob-analysis work in Node.
```
Comment (SidharthBansal) [inline src/modules/BlobAnalysis/BlobAnalysis.js]: .

File: src/modules/BlobAnalysis/Module.js [hunk 5]
```diff
@@ -10,7 +10,7 @@ module.exports = function BlobAnalysis(options, UI){
   
     var step = this;
 
-    var priorStep = this.getStep(-1); // get the previous step to process it
+    var priorStep = this.getStep(-1); // Get the previous step to process it.
```
Comment (SidharthBansal) [inline src/modules/BlobAnalysis/Module.js]: .

**Comment by SidharthBansal** [comment]
Kindly remove two .
Let's complete all your prs today. Only . errors are remaining I guess

**Comment by HarshKhandeparkar** [comment]
Okay. Ty.

File: src/modules/BlobAnalysis/BlobAnalysis.js (hunk 4 above)
Comment (HarshKhandeparkar) [reply src/modules/BlobAnalysis/BlobAnalysis.js]:
```suggestion
  var $ = require('jquery'); // To make Blob-analysis work in Node
```

File: src/modules/BlobAnalysis/Module.js (hunk 5 above)
Comment (HarshKhandeparkar) [reply src/modules/BlobAnalysis/Module.js]:
```suggestion
    var priorStep = this.getStep(-1); // Get the previous step to process it
```

**Comment by SidharthBansal** [comment]
Please do as per other comments. Thanks

**Comment by jywarren** [comment]
Fantastic work! Thanks!!

**Pull request merged**